package.json

```diff
@@ -118,7 +118,7 @@
     "typescript": "4.5.4"
   },
   "engines": {
-    "node": ">=16.0.0 <17.0.0"
+    "node": ">=16.0.0 <17.4.0"
```

Comment:
What is the reason to bump this?

Reply:
I'm locally working with `17.4.0` and this check would block local development.